resources/js/Components/Form.vue

```diff
@@ -50,6 +50,7 @@ const submitForm = async (FormData: FormData, form$: any) => {
                 form$.clear();
                 form$.reset();
             }
+            form$.messageBag.append('Password successfully changed', 'success')
```

Comment:
The string has to be generic or come from the backend somehow. but thats a good start. maybe just a saved. by default and if a certain prop is present on the response it is taken.

Reply:
Absolutely! This is just a preliminary example of how success messages can be displayed. Next week, we can explore making this dynamic during our call. I'm looking forward to discussing it further!